std_msgs/Header header

geometry_msgs/Vector3 loc   # loc.x (m)
geometry_msgs/Vector3 rot   # rot.x = rotation.roll (deg)
geometry_msgs/Vector3 vel   # vel.x (m/s)
geometry_msgs/Vector3 ang_vel   # ang_vel.x (deg/s)



